runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -1227,6 +1229,7 @@ class CompilationInfo
 
 #if defined(J9VM_OPT_JITSERVER)
    ClientSessionHT               *_clientSessionHT; // JITServer hashtable that holds session information about JITClients
+   PersistentUnorderedMap<J9ROMClass*, JITServerHelpers::ClassInfoTuple> _romJ9ClassMap;
```

Comment:
Why do we need `ClassInfoTuple`? Maybe all we need here is an `unordered_set` instead of an `unordered_map`

Reply:
Maybe add some comments with the purpose of this hashtable.